app/decorators/models/solidus_easypost/spree/stock/estimator_decorator.rb

```diff
@@ -4,26 +4,36 @@ module SolidusEasypost
   module Spree
     module Stock
       module EstimatorDecorator
-
         def shipping_rates(package, frontend_only = true)
           if ::Spree::Easypost::Config.api_enabled && !package.stock_location.is_digital?
             shipment = package.easypost_shipment
             easypost_rates = shipment.rates.sort_by { |r| r.rate.to_i }
 
             shipping_rates = calculate_shipping_rates(package)
-            shipping_rates.select! { |rate| rate.shipping_method.available_to_users? && rate.shipping_method.is_easypost === false } if frontend_only
+            if frontend_only
+              shipping_rates.select! do |rate|
+                rate.shipping_method.available_to_users? &&
+                  rate.shipping_method.is_easypost == false
+              end
+            end
 
             if easypost_rates.any?
+              eligible_methods = eligible_shipping_methods(package)
```

Comment:
Not rejecting rates for easypost shipping methods that the package isn't eligible for by way of its contents' shipping categories seems like a mistake, right?